hack/import-calendar.py

```diff
@@ -1,9 +1,10 @@
-#!/usr/bin/python
+#!/usr/bin/python3.7
```

Comment:
```suggestion
#!/usr/bin/python3
```

This should be easier to fulfil on other setups, e.g. on Ubuntu 20.04:

```cli
❯ ls -l /usr/bin/python3
lrwxrwxrwx 1 root root 9 Mär 13  2020 /usr/bin/python3 -> python3.8
~ on ☁️  daniel@weave.works(europe-west1) 
❯ 
```

Reply:
This might fix the link checker action as well.

Reply:
I'm bit confused. I did this via a commit:

```
#!/usr/bin/python3
import sys
print(sys.version)
```
Commented out rest of the script. I got `3.5.7` from Netlify build log. Did you upgrade it elsewhere?

Reply:
This is strange...... so maybe indeed we need to find a way to specifically point to 3.7 somehow. I thought that's what `runtime.txt` would do: https://docs.netlify.com/configure-builds/manage-dependencies/#python

Maybe let's set `PYTHON_VERSION` in the `Makefile` to be sure?

Reply:
At some stage we should look into moving from Ubuntu 16.04 to Ubuntu 20.04 (just noticed that we're not on the newest yet). Will file a separate issue.

Reply:
I set `PYTHON_VERSION := 3.7` in Makefile. Still getting `3.5` from `print(sys.version)`